processing/src/main/java/org/apache/carbondata/lcm/status/SegmentStatusManager.java

```diff
@@ -97,15 +91,14 @@ public long getTableStatusLastModifiedTime() throws IOException {
    * @return
    * @throws IOException
    */
-  public ValidAndInvalidSegmentsInfo getValidAndInvalidSegments() throws IOException {
+  public static ValidSegmentsInfo getValidSegments(AbsoluteTableIdentifier identifier)
```

Comment:
In old code we are getting both valid and invalid segments at once time by reading status file one time. Now we need to read twice for getting the same. Is there any reason of changing the old code?